<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerationInstance</class>
 <widget class="QDialog" name="GenerationInstance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="nbrInstanceslabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre d'instances à générer :</string>
   </property>
  </widget>
  <widget class="QLabel" name="nbrJobsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de jobs :</string>
   </property>
  </widget>
  <widget class="QLabel" name="nbrRessourcesLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de ressources :</string>
   </property>
  </widget>
  <widget class="QLabel" name="nbrMachinesLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de machines :</string>
   </property>
  </widget>
  <widget class="QLabel" name="horizonPlanificationLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizon de planification :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nbrJobsLineEdit">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>60</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nbrRessourcesLineEdit">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>90</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nbrMachinesLineEdit">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>120</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nbrInstancesLineEdit">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>150</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="horizonPlanificationLineEdit">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>180</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annulerPushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QComboBox" name="horizonPlanificationComboBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>secondes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>minutes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>heures</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>jours</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Génération d'instances</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
